Sources/Client/Storage/StorageManager.swift

```diff
@@ -9,10 +9,26 @@ class StorageManager {
   /// Whether this is the first time the app has been launched or not.
   public let isFirstLaunch: Bool
   /// The directory to store all of the client's persistent data.
-  public var storageDirectory: URL
+  private(set) var storageDirectory: URL
   
   /// The directory within the storage directory to store the vanilla assets.
   public var vanillaAssetsDirectory: URL
+  
+  enum Cache: String, CaseIterable {
```

Comment:
i don't think this should be an enum, doesn't make sense to me. I think it was better before as a computed property of StorageManager. And it'll only ever have one case. DeltaClient will always access any other resource packs dynamically (not hardcoded).

Reply:
Ah alright, didn't know it would have only ever have one case
Fixing it 👍